packages/tooltip/src/Crosshair.tsx

```diff
@@ -52,15 +51,3 @@ const Crosshair = memo(({ width, height, type, x, y }) => {
 })
 
 Crosshair.displayName = 'Crosshair'
-Crosshair.propTypes = {
-    width: PropTypes.number.isRequired,
-    height: PropTypes.number.isRequired,
-    x: PropTypes.number.isRequired,
-    y: PropTypes.number.isRequired,
-    type: crosshairPropTypes.type.isRequired,
-}
-Crosshair.defaultProps = {
-    type: 'cross',
```

Comment:
Was this lost during the conversion? I don't see a default value for `type` above.